invenio_rest/config.py

```diff
@@ -75,3 +75,54 @@
           'marc21': 'application/marcxml+xml'
        }
 """
+
+REST_CSRF_ENABLED = True
+"""Enable CSRF middleware. (Default: ``True``).
+
+.. note::
+   The CSRF middleware accepts some configuration parameters that are not set
```

Comment:
Does this mean that some params **must** be defined? If yes, I would re-phrase it? E.g.
`
When CSRF check is enabled, the following configuration parameters must be defined as they affect... . They do not come out of the box with a default value.
`